pkg/virt-handler/vm.go

```diff
@@ -1287,6 +1287,10 @@ func (c *VirtualMachineController) calculateLiveStorageMigrationCondition(vmi *v
 		multiCond.addNonMigratableCondition(v1.VirtualMachineInstanceReasonHostDeviceNotMigratable, "VMI uses a PCI host devices")
 	}
 
+	if util.IsVMIVirtiofsEnabled(vmi) {
+		multiCond.addNonMigratableCondition(v1.VirtualMachineInstanceReasonVirtIOFSNotMigratable, "VMI uses virtiofs")
```

Comment:
I don't think this is competently correct. The VM could support storage live migration, but we cannot storage live migrate volumes that are filesystem. Since, now, virtiofs supports live migration, you can live migrated the VM, if the migrated volumes aren't filesystems. There is already a [check for the volumes](https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-controller/watch/volume-migration/volume-migration.go#L125)

Reply:
Makes sense, thanks Alice!